wgpu-core/src/command/render.rs

```diff
@@ -625,6 +625,8 @@ pub enum ColorAttachmentError {
         mip_level: u32,
         depth_or_array_layer: u32,
     },
+    #[error("Color attachment's usage cannot contain {0:?} if StoreOp is {1:?}")]
```

Comment:
It would be a bit more helpful to the user if this said something like this. Maybe make the middle "StoreOp::discard" an argument too? Not sure.

```suggestion
    #[error("Color attachment's usage contains {0:?}. This can only be used with StoreOp::Discard, but the provided store op was {1:?}")]
```